test/agentchat/test_agent_logging.py

```diff
@@ -242,6 +245,14 @@ def test_groupchat_logging(db_connection):
     rows = cur.fetchall()
     assert len(rows) == 3
 
+    # Verify events
+    cur.execute(EVENTS_QUERY)
+    rows = cur.fetchall()
+    json_state = json.loads(rows[0]["json_state"])
+    assert rows[0]["event_name"] == "received_message"
+    assert json_state["message"] == "Can you explain the difference between eigenvalues and singular values again?"
+    assert len(rows) == 15
```

Comment:
just want to double check, will there be more than 15rows if some of the open ai config fails?

Reply:
not sure - maybe it will produce same number unless the error makes it return earlier.